src/controllers/admin/users_roles.js

```diff
@@ -0,0 +1,373 @@
+import usersService from '../../services/users';
+import applicationError from '../../utils/Errors/applicationError';
+import userBadRequest from '../../utils/Errors/badRequestError';
+import notFound from '../../utils/Errors/notFoundRequestError';
+import roleServices from '../../services/roles';
+import accessDenied from '../../utils/Errors/accessDenied';
+import readData from '../../utils/readData';
+
+
+exports.welcome =(req, res) =>{
+    res.status(200).json({ status: 200, message: 'Welcome as an administrator of Barefoot nomad' });
+}
+
+
+
+exports.findThem = async (req, res, next) =>{
```

Comment:
can you use the same covention as the rest of the team `_export.fuctionName_` replace it with **export functionName**